src/state/disconnect-toast-mixin.ts

```diff
@@ -1,37 +1,30 @@
 import { Constructor, LitElement } from "lit-element";
 import { HassBaseEl } from "./hass-base-mixin";
-import { HaToast } from "../components/ha-toast";
-import { computeRTL } from "../common/util/compute_rtl";
+import { fireEvent } from "../common/dom/fire_event";
 
 export default (superClass: Constructor<LitElement & HassBaseEl>) =>
   class extends superClass {
-    private _discToast?: HaToast;
-
     protected firstUpdated(changedProps) {
       super.firstUpdated(changedProps);
       // Need to load in advance because when disconnected, can't dynamically load code.
-      import(/* webpackChunkName: "ha-toast" */ "../components/ha-toast");
+      import(/* webpackChunkName: "notification-manager" */ "../managers/notification-manager");
     }
 
     protected hassReconnected() {
       super.hassReconnected();
-      if (this._discToast) {
-        this._discToast.opened = false;
-      }
+
+      fireEvent(this, "hass-notification", {
+        message: "",
+        duration: 1,
+      });
     }
 
     protected hassDisconnected() {
       super.hassDisconnected();
-      if (!this._discToast) {
-        const el = document.createElement("ha-toast");
-        el.duration = 0;
-        this._discToast = el;
-        this.shadowRoot!.appendChild(el as any);
-      }
-      this._discToast.dir = computeRTL(this.hass!);
-      this._discToast.text = this.hass!.localize(
-        "ui.notification_toast.connection_lost"
-      );
-      this._discToast.opened = true;
+
+      fireEvent(this, "hass-notification", {
```

Comment:
Not 100% sure about this, this toast should never be overridden.

Reply:
I think, that it's behaviour will not change... As the toast would also close when another would open: https://github.com/PolymerElements/paper-toast/blob/master/paper-toast.js#L253-L255